testing/integration/install_test.go

```diff
@@ -32,34 +31,16 @@ func TestInstall(t *testing.T) {
 		// It's not safe to run this test locally as it
 		// installs Elastic Agent.
 		Local: false,
-
-		// Since this test sets the location on the
-		// filesystem where Elastic Agent will be
-		// installed, it's probably safest not to run
-		// it along with other tests.
-		Isolate: true,
 	})
 
 	// Get path to Elastic Agent executable
 	fixture, err := define.NewFixture(t)
 	require.NoError(t, err)
 
-	suite.Run(t, newInstallTestSuite(fixture))
-}
-
-func newInstallTestSuite(fixture *atesting.Fixture) *InstallTestSuite {
-	i := new(InstallTestSuite)
-	i.fixture = fixture
-
-	return i
-}
-
-type InstallTestSuite struct {
```

Comment:
Is there a reason to get rid of the test suite and only have a simple test function ? I thought that the integration test fwk needed some hooks from the testsuite, has this changed ?